packages/utilities/src/focus.ts

```diff
@@ -467,10 +467,23 @@ export function focusAsync(element: HTMLElement | { focus: () => void } | undefi
     if (win) {
       // element.focus() is a no-op if the element is no longer in the DOM, meaning this is always safe
       win.requestAnimationFrame(() => {
-        targetToFocusOnNextRepaint && targetToFocusOnNextRepaint.focus();
+        const focusableElement = targetToFocusOnNextRepaint as HTMLElement | null;
 
         // We are done focusing for this frame, so reset the queued focus element
         targetToFocusOnNextRepaint = undefined;
+
+        if (focusableElement) {
+          if (focusableElement.getAttribute && focusableElement.getAttribute(IS_FOCUSABLE_ATTRIBUTE) === 'true') {
```

Comment:
isn't there a helper for checking if an element is focusable? isTabbable or something.

Reply:
The browser doesn't seem to have any issue applying focus to natively-tabbable elements, just the custom-tabbable ones. Hence why I scoped this to only `IS_FOCUSABLE_ATTRIBUTE`.

Reply:
I meant `isElementTabbable` in utilities.